### PR TITLE
php yii: add page

### PR DESCRIPTION
**Pull request by ImLiam**
This adds a page for [Yii Framework's command line interface](https://www.yiiframework.com/doc/guide/2.0/en/tutorial-console)

-----

- [x] The page (if new), does not already exist in the repo.

- [x] The page (if new), has been added to the correct platform folder:  
      `common/` if it's common to all platforms, `linux/` if it's Linux-specific, and so on.

- [x] The page has 8 or fewer examples.

- [x] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [x] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

File: pages/common/php-yii.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# php yii
+
+> Yii Framework's command line interface.
+
+- Display a list of all available commands:
+
+`php yii {{help}}`
```
Comment (sbrl) [inline pages/common/php-yii.md]: Wouldn't this execute a file called `yii` in the current directory? I assume that a dash `-` is missing here, or the `php` bit is extraneous, or something else?

Not familiar with the _Yii Framework_, so I could be wrong.

File: pages/common/php-yii.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+- Start PHP's built-in web server for the current Yii application:
```
Comment (sbrl) [inline pages/common/php-yii.md]: What's the difference between this and `php -a`?

File: pages/common/php-yii.md (hunk 1 above)
Comment (ImLiam) [reply pages/common/php-yii.md]: That is correct. Mentioned more in the [php artisan PR](https://github.com/tldr-pages/tldr/pull/2346#discussion_r219605667)

File: pages/common/php-yii.md (hunk 2 above)
Comment (ImLiam) [reply pages/common/php-yii.md]: This command serves the application while bootstrapping the framework and using appropriate configuration values where necessary.

File: pages/common/php-yii.md (hunk 1 above)
Comment (sbrl) [reply pages/common/php-yii.md]: Cool, thanks

File: pages/common/php-yii.md (hunk 2 above)
Comment (sbrl) [reply pages/common/php-yii.md]: Ah, ok

**Pull request merged**